<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Historyjka</class>
 <widget class="QDialog" name="Historyjka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="podglad">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>600</width>
     <height>340</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="Wybor_lista">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ButOpen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Otwórz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButDelete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButKill">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyczyść Historie</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
